docs/openstack/node-selector.md

```diff
@@ -0,0 +1,144 @@
+Node Selector
```

Comment:
This doc looks like it applies the same for greenfield deployments as well? Maybe it should eventually live in some "common docs place" and be referenced from Adoption docs.

Reply:
Yes, you are right, a lot of the stuff this doc adds needs to go into a generic doc that applies for both, but I thought I would add it first here and then figure out what we are doing for greenfield.